src/python/director/screengrabberpanel.py

```diff
@@ -27,9 +27,11 @@ def __init__(self, widgets):
 
 
 class ScreenGrabberPanel(object):
+    # HACK
+    instance = None
 
     def __init__(self, view):
-
+        ScreenGrabberPanel.instance = self
```

Comment:
Oops, didn't realize there is already a GUI class, the `LcmLogPlayerGui` is perfect.  Just pass the `screenGrabberPanel` variable into the ctor of the `LcmLogPlayerGui`.